### PR TITLE
Update to support Django-1.9

### PR DESCRIPTION
**Pull request by nailgun**
Can you check and merge this please. We are upgrading our project to Django-1.9, but we depend on django-affiliate which doesn't support it.

### CODE REVIEW

File: affiliate/templatetags/affiliate_tags.py [hunk 1]
```diff
@@ -47,10 +48,3 @@ def url_aff(parser, token):
                 args.append(parser.compile_filter(value))
 
     return AffiliateURLNode(viewname, args, kwargs, asvar)
-
-
-@register.simple_tag(takes_context=True)
```
Comment (nailgun) [inline affiliate/templatetags/affiliate_tags.py:52]: Looks like this is obsolete tag. Called `render_img` function doesn't exist anywhere.

**Comment by st4lk** [comment]
Thanks!

**Comment by st4lk** [comment]
Now in pypi [v0.4.0](https://pypi.python.org/pypi/django-affiliate/0.4.0)

**Comment by nailgun** [comment]
Thanks a lot!

**Pull request merged**